src/burp/mvol.cpp

```diff
@@ -203,7 +203,7 @@ static void calc_hash(Firebird::string& valid, Firebird::IDbCryptPlugin* plugin)
 //____________________________________________________________
 //
 //
-static Firebird::IKeyHolderPlugin* mvol_get_holder(BurpGlobals* tdgbl, Firebird::RefPtr<const Config>& config)
+static Firebird::IKeyHolderPlugin* mvol_get_holder(BurpGlobals* tdgbl, Firebird::RefPtr<const Firebird::Config>& config)
```

Comment:
You should not qualify with `Firebird::` when you are already in the Firebird namespace.

Reply:
Yes but this line is not in Firebird namespace and all the rest of identifiers in it are qualified.

Reply:
Sorry, I've meant in src/common/classes/GetPlugins.h. (but I didn't reviewed things completelly)

Reply:
>  I've meant in src/common/classes/GetPlugins.h.

Yes this is the only place where such unnecessary qualification has to be done because of https://github.com/FirebirdSQL/firebird/pull/280#discussion_r475358044 #